ingestion-edge/constraints.txt

```diff
@@ -0,0 +1,82 @@
+aiofiles==0.4.0 \
```

Comment:
What is the difference between requirements and constraints? When I've seen multiple requirements files before they've been named `requirements-dev.txt`, etc. Could this be named `requirements-<something>.txt` to make it more obvious it's a pip requirements file?

Reply:
`requirements.txt` is the packages to install, `constraints.txt` constrains versions that can be installed for dependencies.

It doesn't make sense to name it `requirements-<something>.txt` imo, because it's not a pip requirements file.

I would be okay with moving both files into a directory, something like `requirements/constraints.txt` and `requirements/defaults.txt`. I don't particularly like `defaults.txt` but I can't think of a better name.